config/vmdb.tmpl.yml

```diff
@@ -313,6 +313,7 @@ workers:
         :capacity: 50
         :amqp_port: 5672
         :amqp_heartbeat: 30
+        :ceilometer_enabled: true
```

Comment:
if, if we are doing the plugin priority and auto checking for availability, I think we don't need this now